### DISCUSSION TITLE
Multi-GPU

### DISCUSSION THREAD

**Discussion by Vakkaria** [prasanthrangan/hyprdots#1760] [Q&A] [answered]
Are there any package to handle multi-GPU? Because i have intel+nvidia and it already knows how to use both together

**Comment by kRHYME7** [comment]
see Hyprland wiki

**Reply by Vakkaria** [reply]
It's using both GPUs by default... It's interesting, because if i will only use nvidia, hyprland will be broken for my case. So i don't know why...

**Reply by kRHYME7** [reply]
system info?

**Reply by Vakkaria** [reply]
Well,
CPU: intel i5-12500H
card0: Iris Xe Graphics
card1: RTX 3050M

Intel drivers:
```bash
$ pacman -Qs intel
local/intel-ucode 20240813-2
    Microcode update files for Intel CPUs
local/libmfx 23.2.2-3
    Intel Media SDK dispatcher library
local/libvpl 2.12.0-1
    Intel Video Processing Library
local/onetbb 2021.13.0-1
    High level abstract threading library (oneAPI Threading Building Blocks)
local/vulkan-intel 1:24.1.6-1
    Open-source Vulkan driver for Intel GPUs
```
    
NVIDIA drivers:
```bash
$ pacman -Qs nvidia
local/egl-wayland 4:1.1.13.1-1
    EGLStream-based Wayland external platform
local/libvdpau 1.5-3
    Nvidia VDPAU library
local/nvidia-open-dkms 555.58.02-16
    NVIDIA open kernel modules
local/nvidia-utils 555.58.02-1
    NVIDIA drivers utilities
```

I have black screen after sddm login.
Result of `echo $DISPLAY` is empty string, and i can't export because: "x not found".

**Reply by kRHYME7** [reply, marked as answer]
done this?

https://wiki.hyprland.org/Nvidia/

**Reply by Vakkaria** [reply]
It helps, thanks a lot

**Reply by sitolam** [reply]
> It helps, thanks a lot

What did you do to fix it because on the wiki page they say that you need to do all sort of stuff like installing nvidia drivers and so on but I thought that was already done in hyprdots?

**Comment by Vakkaria** [comment]
Alright, one more question. Can i use supergfxctl and asusctl for change mode to MUX Switch?

**Reply by kRHYME7** [reply]
Are you using an Asus laptop? If it's ROG I guess you should. we got a patch for this too. ` Hyde inject asus_patch ` to prepare your repo and have the services running. This is pretty much essential as some of their drivers are proprietary.

But please check your device first. Search Asus on archlinux you'll see some  posts about that

**Reply by Vakkaria** [reply]
I use asus tuf gaming, I saw somewhere that arch (or hyprland) doesn't recommend to use optimus manager.

**Reply by kRHYME7** [reply]
https://asus-linux.org/guides/arch-guide/
https://wiki.archlinux.org/title/Laptop/ASUS

Should help

**Reply by Vakkaria** [reply]
I've been there. I'm just not sure if it will work properly on hyprland, because there have been things that don't work on hyprland but on arch. Anyway, thanks for help!
